static/app/components/version.tsx

```diff
@@ -117,7 +117,7 @@ const Version = ({
 
       <Clipboard value={version}>
         <TooltipClipboardIconWrapper>
-          <IconCopy size="xs" color="white" />
+          <IconCopy size="xs" color="textColor" />
```

Comment:
Think we could just remove the `color` prop entirely. The icon will default to the current text color, which is `textColor`.

Reply:
yep, that works!